lib/net/modules/dns.toit

```diff
@@ -57,7 +57,60 @@ DEFAULT_CLIENT ::= DnsClient [
 // are separated by slash (/) in the key.
 ALL_CLIENTS_ ::= {DEFAULT_CLIENT.servers_.join "/": DEFAULT_CLIENT}
```

Comment:
I'm not sure I follow how this ALL_CLIENTS_ stuff works. You're adding new clients to this on demand, but you seed it with a pretty specific single entry which is:

8.8.8.8/1.1.1.1/8.8.4.4/1.0.0.1 -> DEFAULT_CLIENT

Does anyone ever ask for that?